### PR TITLE
Clarify ALLOW-AXFR-FROM.

### PR DESCRIPTION
**Pull request by GaveUp**

### CODE REVIEW

**Comment by pieterlexis** [comment]
You might want to add "to your `pdns.conf`" to the sentence as well for even better clarity :smile:

**Comment by GaveUp** [comment]
Left off the file name since the proper place in some setups is not `pdns.conf`. For example, Ubuntu packages are set up so the changes would go in `pdns.d/pdns.local.conf`.

File: docs/markdown/authoritative/domainmetadata.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ insert into domainmetadata (domain_id, kind, content) values (7,'ALLOW-AXFR-FROM
 insert into domainmetadata (domain_id, kind, content) values (7,'ALLOW-AXFR-FROM','2001:db8::/48');
 ```
 
+To dissallow all IP's, except those explicitly allowed by domainmetadata records, add
```
Comment (Habbie) [inline docs/markdown/authoritative/domainmetadata.md]: disallow, IPs

**Comment by Habbie** [comment]
Still, please clarify that allow-axfr-ips goes into the config, not the meta table. Thanks!

**Pull request merged**